test/src/gotrue_admin_mfa_api_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:dart_jsonwebtoken/dart_jsonwebtoken.dart';
+import 'package:dotenv/dotenv.dart' show env, load;
+import 'package:gotrue/gotrue.dart';
+import 'package:http/http.dart' as http;
+import 'package:test/test.dart';
+
+void main() {
+  load(); // Load env variables from .env file
+
+  final gotrueUrl = env['GOTRUE_URL'] ?? 'http://localhost:9998';
+  final serviceRoleToken = JWT(
+    {'role': 'service_role'},
+  ).sign(
+    SecretKey(
+        env['GOTRUE_JWT_SECRET'] ?? '37c304f8-51aa-419a-a1af-06154e63707a'),
+  );
+
+  final userId2 = "28bc7a4e-c095-4573-93dc-e0be29bada97";
+  final factorId2 = "2d3aa138-da96-4aea-8217-af07daa6b82d";
```

Comment:
Comments might help anyone looking at this code in the future!
```suggestion
  /// User ID of the user with verified factor
  const userId2 = "28bc7a4e-c095-4573-93dc-e0be29bada97";
  /// Factor ID of `userId2`
  const factorId2 = "2d3aa138-da96-4aea-8217-af07daa6b82d";
```

Reply:
Sorry, absolutely!